### DISCUSSION TITLE
Additional events for basic blocks

### DISCUSSION THREAD

**Discussion by kemeris2000** [lowdefy/lowdefy#1707] [Ideas and feature requests]
It would be great to have onExit (onFocus or onBlur) event for TextInput block. One use case would be custom input validation, for example to check if user's provided email is already in database before submitting data.

**Comment by Gervwyk** [comment]
Hi @kemeris2000 
Thanks for the feedback. I've added these events to the input blocks. Should be in the next release.
https://github.com/lowdefy/lowdefy/pull/1709

**Reply by kemeris2000** [reply]
Great, thank you.

**Comment by kemeris2000** [comment]
onBlur event works, but it gets triggered when TextInput lose focus and also when receive focus. Maybe something forces TextInput lose and instantly receive focus again, not sure about that yet.
